### PR TITLE
cmder colors

### PR DESCRIPTION
**Pull request by kolmogorov42**
Back with a new pull request and screenshots.

This time, I recreated the theme colors based on the image in the README page, since color codes seemed to be inconsistent across files. I took advantage of the ConEmu feature that allows users to pick one of 16 base colors as default text color, and another as default BG color.

I had to change one small option in your file, as to prevent cmder from starting in Powershell mode by default, which caused an error message and is probably not what the user wants.

In addition, I modified your LS_COLORS a bit as a way to prevent directories and other kinds of items from being shown on a "black" background, which is not the same as the default BG color I picked, following your design. So I just stripped all the "40" codes I could find, though this might not be ideal (you might want specific kinds of items to really be displayed on black, rather than the terminal's bg color).

There's a whole bunch of other things I'd like to discuss about LS_COLORS, but that might be the subject of another PR.

#### Screens before PR
conemu64_2017-11-19_15-32-43
conemu64_2017-11-19_15-37-32

#### After PR
conemu64_2017-11-19_16-01-24
conemu64_2017-11-19_16-06-34

Screens show the Dark theme, but my xml file allows the user to pick either.

Screens have been taken from a fresh cmder-mini install, showing Linux from a Vagrant install equipped with oh-my-zsh and materialshell.zsh.

### CODE REVIEW

**Comment by carloscuesta** [comment]
Going to review it! @kolmogorov42

File: materialshell.zsh [hunk 1]
```diff
@@ -48,5 +48,5 @@ ZSH_THEME_GIT_PROMPT_SHA_AFTER="%{$reset_color%}]"
 
 # LS colors, made with http://geoff.greer.fm/lscolors/
 export LSCOLORS="exfxcxdxbxegedabagacad"
-export LS_COLORS='di=34;40:ln=35;40:so=32;40:pi=33;40:ex=31;40:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
+export LS_COLORS='di=34:ln=35:so=32:pi=33:ex=31:bd=34;46:cd=34;43:su=0;41:sg=0;46:tw=0;42:ow=0;43:'
```
Comment (carloscuesta) [inline materialshell.zsh]: Why do you changed the `LS_COLORS`?

**Review by carloscuesta — APPROVED** [review]
Looks great! 🚀

Thanks 🎉

**Comment by kolmogorov42** [comment]
Cheers! 🍻
As for LS_COLORS, do change it back if you think I've been messing too much with the backgrounds!

**Pull request merged**